conf/nginx/nginx.conf.web.https.default.template

```diff
@@ -58,6 +58,22 @@ server
         set $login_upstream    ${web.upstream.webclient.target};
     }
 
+    # if to loginOp is logout, all the requests should be routed to login server so that updated assets can be loaded
+    if ($arg_loginOp = "logout") {
+        set $login_upstream    ${web.upstream.login.target};
+    }
+
+    # Same as above for relogin loginOp
+    if ($arg_loginOp = "relogin") {
+        set $login_upstream    ${web.upstream.login.target};
+    }
+
+    # the mobile client has been discontinued in Zimbra9, hence if the client is mobile, rerouting the requests to user's
+    # server - which could be an old one and not yet upgraded
+    if ($arg_client = "mobile") {
+        set $login_upstream    https://zimbra_ssl_webclient;
```

Comment:
Looks Ok for now,
but there can be scenarios where we are making more than one if conditions to be executed with a need for login_upstream set to different targets, like in example `loginOp=logout&client=mobile`.